flexible-classroom/develop/supply-course-materials.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "Supply course materials"
+sidebar_position: 6
+type: docs
+description: >
+    Supply learning materials for a course. 
+---
+
+
```

Comment:
We updated this article today based on PM's feedback, please help update accordingly.

Reply:
Yep, you can refer to https://docs.agora.io/en/agora-class/agora_class_files?platform=All%20Platforms.

Reply:
You are free to update the content in another branch and merge back.

Reply:
Please supply the source and I'll do it.

Reply:
> Please supply the source and I'll do it. This isn't the source, it's an HTML page :-).

[Supply learning materials for a course.md](https://github.com/AgoraIO/Docs-Source/files/9621905/Supply.learning.materials.for.a.course.md)